src/data-workspace/use-close-right-hand-panel-on-selection-change.js

```diff
@@ -3,16 +3,16 @@ import { useParams } from 'react-router-dom'
 import { useRightHandPanelContext } from '../right-hand-panel/index.js'
 
 export default function useCloseRightHandPanelOnSelectionChange() {
-    const { show } = useRightHandPanelContext()
+    const { hide } = useRightHandPanelContext()
     const searchParams = useParams()
 
     useEffect(
         () => {
-            show('')
+            hide()
         },
         // search params are part of the dependency array as we need to hide
         // the sidebar when the selection changes. There's no need to use the
         // search params in the useEffect though
-        [searchParams, show]
+        [searchParams, hide]
```

Comment:
I realize this hook was using `useParams`. Other hooks do this differently, see `src/context-selection/section-filter-selector-bar-item/use-on-dependent-params-change.js`. This way the user can change the section filter without closing the right hand panel. Plus we can add any query param without having to worry about the sidebar being closed

Reply:
updated